input/cql/HIVElements.cql

```diff
@@ -11,9 +11,9 @@ include WHOElements called WE
 
 include HIVConcepts called Concepts
 include HIVConceptsCustom called CustomConcepts
-//include HIVCommon called Common
+include HIVCommon called Common
 
-parameter "Measurement Period" Interval<Date> default Interval[@2020-01-01, @2020-12-31]
+// parameter "Measurement Period" Interval<Date> default Interval[@2020-01-01, @2020-12-31]
 parameter "Measurement Date" Date default @2020-01-01
 parameter "Testing Interval" System.Quantity default 3 months
```

Comment:
This parameter is a little weird. I think 3 month re-testing is standard for clients on PrEP, but, e.g., for clients on ART who aren't experiencing treatment failure, its generally annual testing and the WHO recommendations are basically annual retests for people at high risk with 3-6 month re-tests "based on individual risk factors".

Maybe it could be renamed?